overlays/haskell.nix

```diff
@@ -492,13 +495,20 @@ final: prev: {
                   then null
                   else
                     let package = package' // { recurseForDerivations = false; };
-                    in package' // {
+                    in package' // rec {
                       components = final.lib.mapAttrs (n: v:
                         if n == "library" || n == "all"
                           then v // { inherit project package; }
                           else final.lib.mapAttrs (_: c: c // { inherit project package; }) v
                       ) package'.components;
                       inherit project;
+
+                      coverageReport = haskellLib.coverageReport {
```

Comment:
Ditto? although this one is more complex...

Reply:
Addressed these here: https://github.com/input-output-hk/haskell.nix/pull/762/commits/29ac8934f18dce96b7c8219577d5586117827808